sql/hive-thriftserver/pom.xml

```diff
@@ -41,11 +41,8 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.apache.spark</groupId>
-      <artifactId>spark-core_${scala.binary.version}</artifactId>
-      <version>${project.version}</version>
-      <type>test-jar</type>
-      <scope>test</scope>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlet</artifactId>
```

Comment:
Yes, the Hive Thrift Server module does directly import jetty classes, and so should add a direct compile time dependency on it as you've done here. It happens to work out fine in most scenarios because it's a transitive dependency but it should be tightened up especially if it resolves an occasional failure like this. It's orthogonal to this change though. It could be a simple separate PR if you're willing.